### PR TITLE
docs update - on-prem builder authentication

### PR DESCRIPTION
**Pull request by ericcalabretta**
update on-prem builder instructions to clarify currently supported authentication options. Clarify a combined install vs installing on a different host.

No code change, docs update only. 

Signed-off-by: ericcalabretta <eric.calabretta@gmail.com>

### CODE REVIEW

File: components/docs-chef-io/content/automate/on_prem_builder.md [hunk 1]
```diff
@@ -60,6 +59,11 @@ Chef Automate and Chef Habitat Builder require:
 * The shell that starts Chef Automate should have a max open files setting of at least 65535
 * Run the installation and bootstrapping procedures as the superuser or use `sudo` at the start of each command.
 
+### Supported Topologies
+
+* Chef Automate and Chef Habitat Builder on the same host
```
Comment (susanev) [inline components/docs-chef-io/content/automate/on_prem_builder.md:64]: so this is easier to scan, might be good to restructure so `same host` and `different hosts` are at the beginning of the lines

**Review by kagarmoe — APPROVED** [review]

**Comment by ericcalabretta** [comment]
Where's the DCO missing? I signed off on the original commits.

**Comment by mjingle** [comment]
Hi @ericcalabretta, because this is a PR from a remote repo, you will have to make the changes to this PR. A suggested edit without a DCO was added on this PR after your original submission. If you could rebase this PR and add your DCO then, that should help get this PR going 🟢  for both the DCO and the pipeline issue. Thanks!

File: components/docs-chef-io/content/automate/on_prem_builder.md [hunk 2]
```diff
@@ -84,9 +88,11 @@ If you are deploying Chef Habitat Builder with Chef Automate in an airgapped env
 
 You can deploy Chef Habitat Builder either with a full Chef Automate installation or with the Chef Automate auth stack only.
 
+When deploying Chef Habitat Builder on a the same host as Chef Automate authentication to Chef Automate will be configured as part of the deployment.
```
Comment (msorens) [inline components/docs-chef-io/content/automate/on_prem_builder.md:91]:
```suggestion
When deploying Chef Habitat Builder on the same host as Chef Automate authentication to Chef Automate will be configured as part of the deployment. 
```

File: components/docs-chef-io/content/automate/on_prem_builder.md [hunk 3]
```diff
@@ -97,7 +103,10 @@ Accept the license with `y`.
 
 #### Deploy Chef Habitat Builder with Chef Automate Auth
 
-To deploy Chef Habitat Builder with the Chef Automate UI and Auth services only, specify the `builder` product on the command line.
+To deploy Chef Habitat Builder on a different host than Chef Automate, specify only the `builder` product on the command line. Chef Automate UI and Auth services only will also be deployed to support authentication.
```
Comment (msorens) [inline components/docs-chef-io/content/automate/on_prem_builder.md:106]: I think this clarifies your intent, but I was not quite sure what that intent was.
```suggestion
To deploy Chef Habitat Builder on a different host than Chef Automate, specify only the `builder` product on the command line. Note that Chef Automate UI and Auth services will also be deployed to support authentication. 
```

**Comment by mjingle** [comment]
@ericcalabretta I made sure your PR's contents was incorporated in https://github.com/chef/automate/pull/4494! Thank you for working on this!!!

**Pull request closed without merging**